homeassistant/components/esphome/manager.py

```diff
@@ -280,42 +280,44 @@ async def _send_home_assistant_state(
 
         await self.cli.send_home_assistant_state(entity_id, attribute, str(send_state))
```

Comment:
This doesn't actually await anything so it could be changed to be a normal function